Makefile

```diff
@@ -374,11 +374,11 @@ $(ARCHIVE): $(ARCHIVE).tmp
 # instead of scripts/ls-files.sh once Git v2.11 is widely deployed.
 .INTERMEDIATE: $(ARCHIVE).tmp
 $(ARCHIVE).tmp: ARCHIVE_BASE = cockroach-$(shell cat .buildinfo/tag)
-$(ARCHIVE).tmp: .buildinfo/tag .buildinfo/rev .buildinfo/basebranch
+$(ARCHIVE).tmp: $(GOBINDATA_TARGET) $(SQLPARSER_TARGETS) .buildinfo/tag .buildinfo/rev .buildinfo/basebranch
```

Comment:
Use $(BUILDINFO) here while you're at it.

Reply:
Done.